### DISCUSSION TITLE
I found a great site for parametrics of many different items

### DISCUSSION THREAD

**Discussion by ReggiePye** [arkypita/LaserGRBL#2426] [General]
https://box.laserbiz.ru/?language=en
This site has so many different generators for making SVG files ready to use in LaserGRBL

**Comment by Hostaler** [comment]
Thanks here you have another very easy to use.

https://boxes.hackerspace-bamberg.de/

**Reply by ReggiePye** [reply]
> Thanks here you have another very easy to use.
> 
> https://boxes.hackerspace-bamberg.de/

Even better, thanks

**Reply by arkypita** [reply]
Wow
